src/common/i18n/translations/en.ts

```diff
@@ -0,0 +1,104 @@
+  loginScanCard: {
+    loginWithQR:
+      "Please log in with your unique QR code provided by your supervisor",
+    scanToLogin: "Scan to login",
+    needHelp: "Need help?",
+    askQuestion: "Ask a question"
+  },
+  loginMobileNumberCard: {
+    enterMobileNumber:
+      "Please enter your mobile phone number to receive a one-time password.",
+    mobileNumber: "Mobile phone number",
+    sendOtp: "Send OTP"
+  },
+  loginOTPCard: {
+    sendingOtp: "We're sending you the one-time password...",
+    otp: "OTP",
+    resendIn: "Resend in %{ss}s",
+    submit: "Submit",
+    resend: "Resend"
+  },
+  collectCustomerDetailsScreen: {
+    checkEligibleItems: "Check the number of item(s) eligible for redemption",
+    scanIdentification: "Scan identification",
+    or: "Or",
+    enterIdNumber: "Enter ID Number",
+    check: "Check"
+  },
+  idScanner: {
+    enterIdManually: "Enter ID manually",
+    enterManually: "Enter manually",
+    enterVoucherId: "Enter voucher ID",
+    scanToCheck: "Scan to check validity"
+  },
+  navigationDrawer: {
+    logout: "Logout",
+    addCampaign: "Add Campaign",
+    selectCampaign: "Select campaign",
+    changeChampaign: "Change Campaign",
+    helpSupport: "Help & Support",
+    termsOfUse: "Terms of use",
+    privacyStatement: "Privacy Statement",
+    reportVulnerability: "Report Vulnerability"
+  },
+  customerQuotaScreen: {
+    campaignExpiredOn: "Expired on %{dateTime}",
+    campaignValidTo: "Valid till %{dateTime}",
+    quotaRedeemedLimitReachedIDNumber: "Identification number(s)",
+    quotaLimitMax: "Max",
+    quotaAppealCancel: "Cancel",
+    quotaButtonCheckout: "Checkout",
+    quotaButtonAdd: "+ Add",
+    quotaAddId: "Add another ID to combine customer quotas",
+    quotaIdentifierButtonScan: "Scan",
+    quotaScanButtonBack: "Back",
+    quotaCheck: "Checking"
+  },
+  merchantFlowScreen: {
+    quotaButtonAddVoucher: "Add voucher",
+    quotaCategoryVouchers: "Voucher(s):",
+    merchantCode: "Merchant Code",
+    seeAll: "See all",
+    nextMerchant: "Next merchant",
+    remove: "Remove"
+  },
+  checkoutSuccessScreen: {
+    redeemed: "Redeemed!",
+    redeemedItems: "Item(s) redeemed:",
+    redeemedNextIdentity: "Next identity",
+    purchased: "Purchased!",
+    purchasedItems: "The following have been purchased:",
+    purchasedNextIdentity: "Next customer",
+    previouslyRedeemedItems: "Item(s) redeemed previously:",
+    previouslyPurchasedItems: "Item(s) purchased previously:",
+    redeemedAgo: "Redeemed %{time} ago",
+    redeemedOn: "Redeemed on %{time}",
+    previouslyRedeemed: "Previously redeemed",
+    showMore: "Show more",
+    showLess: "Show less",
+    today: " for today.",
+    limitReachedRecent: "Limit reached %{time} ago.",
+    limitReachedDate: "Limit reached on %{dateTime}",
+    limitReached: "Limit reached",
+    redeemedLimitReached: "%{quantity} item(s) more till %{date}.",
+    valid: "Valid",
+    complete: "Complete",
+    quantity: " qty"
```

Comment:
Can we leave the prefixed space in the code rather than here, so anyone looking through the code understands that we want to have a space prefix?

Reply:
Same comment for the rest wherever the punctuation is a prefix or suffix. If in the middle I understand we need to keep in the translated copy. Help to take a look pls 🙏